package com.vidio.platform.api;

import com.vidio.android.api.InterceptorConstantKt;
import com.vidio.platform.gateway.responses.VntSessionResource;
import kotlin.Metadata;
import kotlin.Unit;
import moe.banana.jsonapi2.l;
import org.jetbrains.annotations.NotNull;
import org.jetbrains.annotations.Nullable;
import retrofit2.http.GET;
import retrofit2.http.Headers;
import retrofit2.http.POST;
import tb0.c;

@Metadata(d1 = {"\u0000\u001c\n\u0002\u0018\u0002\n\u0002\u0010\u0000\n\u0002\u0010\u0002\n\u0002\b\u0002\n\u0002\u0018\u0002\n\u0002\u0018\u0002\n\u0002\b\u0002\bf\u0018\u00002\u00020\u0001J\u0010\u0010\u0003\u001a\u00020\u0002H§@¢\u0006\u0004\b\u0003\u0010\u0004J\u0016\u0010\u0007\u001a\b\u0012\u0004\u0012\u00020\u00060\u0005H§@¢\u0006\u0004\b\u0007\u0010\u0004¨\u0006\bÀ\u0006\u0003"}, d2 = {"Lcom/vidio/platform/api/VntApi;", "", "", "createSession", "(Ltb0/c;)Ljava/lang/Object;", "Lmoe/banana/jsonapi2/l;", "Lcom/vidio/platform/gateway/responses/VntSessionResource;", "getSession", "shared"}, k = 1, mv = {2, 3, 0}, xi = 48)
/* loaded from: classes6.dex */
public interface VntApi {
    @Headers({InterceptorConstantKt.REQUIRE_AUTH})
    @POST("/vnt/session")
    @Nullable
    Object createSession(@NotNull c<? super Unit> cVar);

    @Headers({InterceptorConstantKt.REQUIRE_AUTH})
    @GET("/vnt/session")
    @Nullable
    Object getSession(@NotNull c<? super l<VntSessionResource>> cVar);
}
